### DISCUSSION TITLE
Option to hide post username

### DISCUSSION THREAD

**Discussion by ghost** [Docile-Alligator/Infinity-For-Reddit#879] [Ideas]
Please grant us option to hide the post username!
Some of us don't need to know if the posts are done by the same or different user on the title, thus removing the clutter!

Thank you!

**Comment by Wladefant** [comment]
Everyone who is interested in this feature, should upvote it.
If it will be implemented, I think the toggle should be somewhere in the hide section and should be called something like: Hide (Poster) Username
